src/styles/utils/_normalize.scss

```
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

```suggestion
h1,
body {
```

Remove default margins from h1 tag also.

---

Ok